src/pymmcore_widgets/mda/_core_mda.py

```diff
@@ -299,6 +330,76 @@ def _enable_widgets(self, enable: bool) -> None:
             elif child is not self.control_btns and hasattr(child, "setEnabled"):
                 child.setEnabled(enable)
 
+    def _show_hcs(self) -> None:
+        """Show or raise the HCS wizard."""
+        self.hcs.raise_() if self.hcs.isVisible() else self.hcs.show()
+
+    def _on_hcs_accepted(self) -> None:
+        """Add the positions from the HCS wizard to the stage positions."""
+        if (plan := self.hcs.value()) is not None:
+            self.stage_positions.setValue(list(plan))
+            self._use_hcs = True
+            self._enable_position_table(False)
+            return
+        self._use_hcs = False
+        self._enable_position_table(True)
+
+    def _enable_positions_editing(self) -> None:
+        dialog = QMessageBox(
+            QMessageBox.Icon.Warning,
+            "Reset HCS",
+            "If you modify the positions, they will be considered as manually added, "
```

Comment:
start with "Positions are currently autogenerated from the HCS Wizard.  Would you like to cast them to a list of stage positions?  Note: you will no longer be able to edit them using the HCS Wizard dialog."

- And then also, add a warning dialog when clicking on "add positions": "This will replace the positions currently stored in the table.  Would you like to proceed?"

- ideally: if the user clicks on the HCS wizard again after the positions are currently being controlled by the wizard, the wizard button should say "update positions"